tests/browser_test.py

```diff
@@ -95,4 +92,52 @@ def test_login_and_submit_review(
     expect(page.locator(".bs-wizard-step.disabled").first).to_contain_text("Approved")
 
 
-# TODO: reviewer test scenario - login to approve pending request
+def test_review_requested_changes(
+    server,
+    page,
+    reviewer_auth,
+    source_bucket,
+    source_collection,
+    make_client,
+    editor_auth,
+):
+    # setup changes to review
+    editor_client = make_client(editor_auth)
+    editor_client.create_record(data={"prop": "val"})
+    editor_client.patch_collection(data={"status": "to-review"})
+
+    # load login page
+    page.goto(f"{server}/admin/")
+    expect(page).to_have_title(re.compile("Remote Settings"))
+
+    # login
+    page.get_by_label("Kinto Account Auth").click()
+    txtUsername = page.get_by_label(re.compile("Username"))
+    txtPassword = page.get_by_label(re.compile("Password"))
+    txtUsername.fill(reviewer_auth[0])
+    txtPassword.fill(reviewer_auth[1])
+    page.get_by_text(re.compile("Sign in using Kinto Account Auth")).click()
+
+    # verify home page loaded
+    expect(page.get_by_text("project_name")).to_be_visible()
+    expect(page.get_by_text("project_version")).to_be_visible()
+    expect(page.get_by_text("http_api_version")).to_be_visible()
+    expect(page.get_by_text("project_docs")).to_be_visible()
```

Comment:
Probably can skip these assertions here since we have them covered in the other test, no?

Reply:
Yeah that's fair, I'll remove the duplicates.